gnomad_qc/analyses/compute_pext.py

```diff
@@ -224,6 +225,20 @@ def main(args):
     ht = pext_res.context_ht.ht().naive_coalesce(5000)
     tx_mt = pext_res.tx_mt.mt()
 
+    # Any tissue below 50 samples will NOT get shown on website in overall pext or as
+    # individual tissue.
+    # Exclude testes and cell lines from overall pext but keep in display as
+    # individual tissues (agreed to take a look at the per tissue pext values to make
+    # sure this makes sense).
+    tissues_to_exclude = get_tissues_to_exclude(
+        tx_mt, reproductive=False, cell_lines=False, min_samples=min_samples
```

Comment:
should these be True now?

Reply:
No, I think we want these still displayed on the browser and in the downloaded file, we just don't want them in the mean. I will add comments

Reply:
Cleaned up and added comments, let me know if it makes sense